src/common/constants.js

```diff
@@ -3981,12 +3981,293 @@ const yesNo = [
+  "Burkina Faso",
+  "Burma",
+  "Burundi",
+  "Cambodia",
+  "Cameroon",
+  "Canada",
+  "Cape Verde",
+  "Cayman Islands",
+  "Central African Republic",
+  "Chad",
+  "Chile",
+  "China",
+  "Colombia",
+  "Comoros",
+  "Congo, Democratic Republic of",
+  "Congo, People's Republic of",
+  "Costa Rica",
+  "Cote d'Ivoire",
+  "Croatia",
+  "Cuba",
+  "Cyprus",
+  "Czech Republic",
+  "Denmark",
+  "Djibouti",
+  "Dominica",
+  "Dominican Republic",
+  "Ecuador",
+  "Egypt",
+  "El Salvador",
+  "England",
+  "Equatorial Guinea",
+  "Eritrea",
+  "Estonia",
+  "Ethiopia",
+  "Fiji",
+  "Finland",
+  "France",
+  "French Guiana",
+  "Gabon",
+  "Gambia",
+  "Georgia",
+  "Germany",
+  "Ghana",
+  "Gibraltar",
+  "Great Britain",
+  "Great Britain and Gibraltar",
+  "Greece",
+  "Greenland",
+  "Grenada",
+  "Guadeloupe",
+  "Guatemala",
+  "Guinea",
+  "Guinea, Republic of Guinea",
+  "Guinea-Bissau",
+  "Guyana",
+  "Haiti",
+  "Honduras",
+  "Hong Kong",
+  "Hungary",
+  "Iceland",
+  "India",
+  "Indonesia",
+  "Iran",
+  "Iraq",
+  "Ireland",
+  "Israel (Jerusalem)",
+  "Israel (Tel Aviv)",
+  "Italy",
+  "Jamaica",
+  "Japan",
+  "Jordan",
+  "Kazakhstan",
+  "Kenya",
+  "Kosovo",
+  "Kuwait",
+  "Kyrgyzstan",
+  "Laos",
+  "Latvia",
+  "Lebanon",
+  "Leeward Islands",
+  "Lesotho",
+  "Liberia",
+  "Libya",
+  "Liechtenstein",
+  "Lithuania",
+  "Luxembourg",
+  "Macao",
+  "Macedonia",
+  "Madagascar",
+  "Malawi",
+  "Malaysia",
+  "Mali",
+  "Malta",
+  "Martinique",
+  "Mauritania",
+  "Mauritius",
+  "Mexico",
+  "Moldavia",
+  "Mongolia",
+  "Montenegro",
+  "Montserrat",
+  "Morocco",
+  "Mozambique",
+  "Namibia",
+  "Nepal",
+  "Netherlands",
+  "Netherlands Antilles",
+  "Nevis",
+  "New Caledonia",
+  "New Zealand",
+  "Nicaragua",
+  "Niger",
+  "Nigeria",
+  "North Korea",
+  "Northern Ireland",
+  "Norway",
+  "Oman",
+  "Pakistan",
+  "Panama",
+  "Papua New Guinea",
+  "Paraguay",
+  "Peru",
+  "Philippines",
+  "Philippines (restricted payments)",
+  "Poland",
+  "Portugal",
+  "Qatar",
+  "Republic of Yemen",
+  "Romania",
+  "Russia",
+  "Rwanda",
+  "Sao-Tome/Principe",
+  "Saudi Arabia",
+  "Scotland",
+  "Senegal",
+  "Serbia",
+  "Serbia/Montenegro",
+  "Seychelles",
+  "Sicily",
+  "Sierra Leone",
+  "Singapore",
+  "Slovakia",
+  "Slovenia",
+  "Somalia",
+  "South Africa",
+  "South Korea",
+  "Spain",
+  "Sri Lanka",
+  "St. Kitts",
+  "St. Lucia",
+  "St. Vincent",
+  "Sudan",
+  "Suriname",
+  "Swaziland",
+  "Sweden",
+  "Switzerland",
+  "Syria",
+  "Taiwan",
+  "Tajikistan",
+  "Tanzania",
+  "Thailand",
+  "Togo",
+  "Trinidad and Tobago",
+  "Tunisia",
+  "Turkey (Adana only)",
+  "Turkey (except Adana)",
+  "Turkmenistan",
+  "USA",
+  "Uganda",
+  "Ukraine",
+  "United Arab Emirates",
+  "United Kingdom",
+  "Uruguay",
+  "Uzbekistan",
+  "Vanuatu",
+  "Venezuela",
+  "Vietnam",
+  "Wales",
+  "Western Samoa",
+  "Yemen Arab Republic",
+  "Zambia",
+  "Zimbabwe"
+];
+
+const pciuStates = [
+  "AK",
```

Comment:
`states.USA` has 62 entries and this has 61. Can we re-use `states.USA` and remove the extra? (Assuming, of course, they map 1:1 otherwise.) If not, we'll have to format this list the same as `states.USA` to give all these options labels for the dropdown menu (and what a pain that'd be!).

Reply:
Oh good catch. Yeah, I'll see if I can clean this up